src/router/index.js

```diff
@@ -0,0 +1,128 @@
+import {getRoutableSignals, flattenConfig} from './utils'
+import urlMapper from 'url-mapper'
+
+let addressbar
+try {
+  addressbar = require('addressbar')
+} catch (e) {
+  addressbar = {
+    pathname: '/',
+    value: '',
+    origin: '',
+    on() {},
+    removeListener() {}
+  }
+}
+
+export default function Router(options = {}) {
```

Comment:
Does that mean user will have to:

1. Install a second dependency, `url-mapper`
2. Intitilize with:

```js
Router({mapper: urlMapper({query: true)})
```

I think we should have a default router. What frameworks has multiple routers? :) If we want to add a different router that should be the whole router, not just an option. So:

```js
import Router from 'cerebral/router'

router: Router({query: true}) // etc.
```
or custom router:
```js
import OtherRouter from 'some-other-cerebral-router'

router: OtherRouter({query: true}) // etc.
```

Though technically it is cool to have different mappers etc., it makes it unnecessarily complicated for someone who just wants a router. I think the router that is integrated now should "just work" as part of Cerebral. If you want to do some edge case routing you just replace `cerebral/router` with something else.